src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/analysis/CartControllerTest.java

```diff
@@ -91,58 +91,58 @@ public void testAddProjectSample() {
 	@Test
 	@Ignore
 	public void testRemoveProjectSamples() {
-		Map<Project, Set<Sample>> selected = new HashMap<>();
-		selected.put(project, samples);
-		controller.addSelected(selected, Locale.ENGLISH);
-
-		Set<Long> subIds = Sets.newHashSet(sampleIds.iterator().next());
-
-		controller.removeProjectSamples(projectId, subIds);
-
-		verify(projectService).read(projectId);
-		verify(sampleService).getSamplesInProject(project, new ArrayList<>(subIds));
-
-		Map<Project, List<Sample>> response = controller.getSelected();
-
-		assertEquals(1, response.keySet().size());
-		Project projectKey = selected.keySet().iterator().next();
-		assertEquals(project, projectKey);
-		for (Sample s : selected.get(projectKey)) {
-			assertFalse(subIds.contains(s.getId()));
-		}
+//		Map<Project, Set<Sample>> selected = new HashMap<>();
```

Comment:
If the test is marked as `@Ignore` is there a reason why you're also commenting it out? Is it because the classes/objects you use have changed and it will be updated in a later pull request?

Reply:
Sounds good 👍